package StacklerleBeraberKullanimi;
//ExceptionPropagations
public class Main {
	
	public static void ununcuFonksiyon() {
		try {
		int a = 12/0;
		}
		catch (ArithmeticException e) {
			// TODO: handle exception
		
		}
		}
	public static void ikinciFonksiyon() {
		ununcuFonksiyon();
	}
	public static void birinciFonksiyon() {
		ikinciFonksiyon();
	}
	
public static void main(String args[]) {
	birinciFonksiyon();
}
}
